File: admin/src/app/models/view-models/bill-detail.ts
import { AutoMap } from "@automapper/classes";
import { BaseViewModel } from "../bases";
import { ProductInBill } from "./product-in-bill";
export class BillDetail extends BaseViewModel {
    @AutoMap()
    id?: number;
    @AutoMap()
    productId?: number;
    @AutoMap()
    product?: ProductInBill;
    @AutoMap()
    price?: number ;
    @AutoMap()
    quantity?: number ;
    @AutoMap()
    discountPrice?: number ;
}

File: App.Models/Entities/BillDetail.cs
using App.Common.Model;

namespace App.Models.Entities
{
    public class BillDetail : BaseEntity<int>
    {
        public virtual Bill Bill { get; set; }
        public int BillId { get; set; }
        public virtual Product Product { get; set; }
        public int ProductId { get; set; }
        public int? Quantity { get; set; }
        public double? Price { get; set; }
        public double? DiscountPrice { get; set; }
    }
}
